src/jimgw/core/jim.py

```
@@ -218,7 +226,7 @@ def sample(
     def get_samples(
         self,
         n_samples: int = 0,
-        rng_key: PRNGKeyArray = jax.random.PRNGKey(21),
+        rng_key: Key = jax.random.key(21),
```

Ruff (0.15.2)

[warning] Do not perform function call `jax.random.key` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)